xbmc/platform/android/network/NetworkAndroid.cpp

```diff
@@ -349,3 +366,25 @@ void CNetworkAndroid::RetrieveInterfaces()
                 n.toString());
   }
 }
+
+void CNetworkAndroid::onAvailable(const CJNINetwork n)
+{
+  CLog::Log(LOGDEBUG, "CNetworkAndroid::onAvailable The default network is now: {}", n.toString());
+
+  CJNIConnectivityManager connman(CXBMCApp::getSystemService(CJNIContext::CONNECTIVITY_SERVICE));
+  CJNILinkProperties lp = connman.getLinkProperties(n);
+
+  if (lp)
+  {
+    CJNINetworkInterface intf = CJNINetworkInterface::getByName(lp.getInterfaceName());
+    if (intf)
+      m_defaultInterface = new CNetworkInterfaceAndroid(n, lp, intf);
+  }
+}
+
+void CNetworkAndroid::onLost(const CJNINetwork n)
+{
+  CLog::Log(LOGDEBUG, "CNetworkAndroid::onLost No default network (the last was: {})",
```

Comment:
Maybe use `CLog::LogF` and discard `CNetworkAndroid::onLost` from the log line?

Reply:
On Android the `LogF()` function does not display the class name, only the function name